src/declarations/stencil-public-runtime.ts

```diff
@@ -1269,6 +1270,7 @@ export namespace JSXBase {
     minLength?: number;
     minlength?: number | string;
     name?: string;
+    onSelect?: (event: Event) => void;
```

Comment:
Can you add the property for `onselect` here as well? (where 'onselect' is all lowercase)
```suggestion
    onSelect?: (event: Event) => void;
    onselect?: (event: Event) => void;
```